App/TopTrumps/TopTrumps.WebApp/Pages/Cards.razor

```diff
@@ -32,6 +32,7 @@
                 <div style="background: url(@card.ImageUrl) no-repeat; background-size: contain; width: 100%; height: 50%; border-radius: 12px;">
                 </div>
                 <h4 class="card-title">@card.Name</h4>
+                <hr class="title">
```

Comment:
Not sure what this looks like but it doesn't quite scan for me. `<hr />` is a horizontal line and it has a class of title. Reading the code I'm not sure what this does.

Reply:
This was an attempt at something that I decided not to do. I must have forgot to remove the html